pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -115,20 +119,14 @@ func (s *USMHTTP2Suite) TestHTTP2DynamicTableCleanup() {
 	matches := PrintableInt(0)
 
 	require.Eventuallyf(t, func() bool {
-		stats := monitor.GetProtocolStats()
-		http2Stats, ok := stats[protocols.HTTP2]
-		if !ok {
-			return false
-		}
-		http2StatsTyped := http2Stats.(map[http.Key]*http.RequestStats)
-		for key, stat := range http2StatsTyped {
+		for key, stat := range getHTTPLikeProtocolStats(monitor, protocols.HTTP2) {
 			if (key.DstPort == http2SrvPort || key.SrcPort == http2SrvPort) && key.Method == http.MethodPost && strings.HasPrefix(key.Path.Content.Get(), "/test") {
 				matches.Add(stat.Data[200].Count)
 			}
 		}
 
-		return matches.Load() == numberOfRequests
-	}, time.Second*10, time.Millisecond*100, "%v != %v", &matches, numberOfRequests)
+		return matches.Load() == usmhttp2.HTTP2TerminatedBatchSize
```

Comment:
Not sure I understand the use of HTTP2TerminatedBatchSize here, why it should be equal to the number of 200? maybe its the naming?

Reply:
we send `usmhttp2.HTTP2TerminatedBatchSize` requests, so we expect to capture `usmhttp2.HTTP2TerminatedBatchSize` endpoints.

Reply:
Oh I see